src/api/app/models/event/request_reviews_done.rb

```diff
@@ -0,0 +1,29 @@
+module Event
+  class RequestReviewsDone < RequestStatechange
+    self.message_bus_routing_key = 'request.reviews_done'
```

Comment:
if this `self` needed? Shouldn't rubocop tell you something? :rage:

Reply:
It's in all other events too - I won't start a difference here. If it's not necessary, it shouldn't be needed for none of the attributes in none of the events, so let's clean this up independently.

Reply:
AFAIK, without the self Ruby couldn't tell if you want to define a new variable or assign it to the instance variable.